streamlit_app.py

```diff
@@ -56,12 +111,23 @@ def draw_state_counties():
         .transform_lookup(lookup='id', from_=alt.LookupData(usda_df, 'FIPS', [selected_usda_feature, 'Name'])) \
         .properties(width=650, height=650)
     st.write(state_map)
```

Comment:
Was this st.write left here on purpose?

Reply:
I didn't really touch this function at all except to have it return the selected state. I think this is your chart with the income data

Reply:
LOL oops my bad. I totally forgot I used the same name. It should be changed anyways with my upcoming PR. Sorry about that

Reply:
oh gotcha, I probably should have not named my map the same tbh, I didn't notice. If you're already changing it though I guess I'll just leave it.